### PR TITLE
DESCW-534 Projects Query Block + minor pattern updates

### PR DESCRIPTION
**Pull request by codewisenate**
Added initial pass at Projects Query block to enable late changes (as of today) to information delivery. Additional work needed to tie output into Actions switching mechanism.

### CODE REVIEW

File: src/Setup.php [hunk 1]
```diff
@@ -305,5 +307,23 @@ public function bcgov_block_theme_page_display() {
...
+	 *
+	 * @return void
+	 */
+	public function bcgov_block_theme_acf_init_block_types() {
+
+		// Check function exists.
+		if ( function_exists( 'acf_register_block_type' ) ) {
 
+			// Load CleanBC Blocks.
+			if ( defined( 'Bcgov\\Theme\\Block\\CLEANBC' ) && CLEANBC ) {
```
Comment (ShawnTurple) [inline src/Setup.php]: I'm not sure why you made CLEANBC all uppercase, i feel like this is going against namepsace naming conventions.
I also feel like the main theme is being polluted with cleanbc, where it should be completely  separate.
I also think you could have benefitted from an autoloader, and could have got away from have includes and require_once all over the place.

**Review by ShawnTurple — APPROVED** [review]

File: src/Setup.php (hunk 1 above)
Comment (codewisenate) [reply src/Setup.php]: This is an error. I am about to push a fix for this check. All I can say is this should not have been pushed.

File: inc/core/acf_blocks/sites/cleanbc/project-query.php [hunk 2]
```diff
@@ -9,12 +9,11 @@
...
 namespace Bcgov\Theme\Block;
 
+// Create id and class attributes allowing for custom "className" and "align" values.
 $elId = 'project-' . $block['id'];
 if ( ! empty( $block['anchor'] ) ) {
     $elId = $block['anchor'];
 }
-
-// Create class attribute allowing for custom "className" and "align" values.
 $className = 'project';
```
Comment (ShawnTurple) [inline inc/core/acf_blocks/sites/cleanbc/project-query.php:17]: mmm, did this pass the phpcs, as this is not a proper php variable name, at least from WordPress's perspective. The context switch between multiple languages can do this :( i also see on line 40 you used the proper naming of $category_detail.

**Review by ShawnTurple — APPROVED** [review]

File: inc/core/acf_blocks/sites/cleanbc/project-query.php (hunk 2 above)
Comment (codewisenate) [reply inc/core/acf_blocks/sites/cleanbc/project-query.php:17]: Yes, passed all phpcs checks. I'll play around with this as I continue to evolve it but yes, I was jumping back and forth between JS and PHP at the time of writing this, so my context was polluted.

**Pull request merged**